packages/app-mobile/components/NoteEditor/NoteEditor.tsx

```diff
@@ -118,6 +118,7 @@ function NoteEditor(props: Props, ref: any) {
 		themeId: props.themeId,
 		themeData: editorTheme(props.themeId),
 		katexEnabled: Setting.value('markdown.plugin.katex') as boolean,
+		spellcheckEnabled: Setting.value('editor.mobile.spellcheckEnabled') as boolean,
```

Comment:
Does `as boolean` actually convert the number to boolean? I don't think it does as it's just information for the compiler. If you want a boolean you should use `!!Setting.value('editor.mobile.spellcheckEnabled')`

Same for the katex setting above.

Reply:
`Setting.value` returns `any`. Adding a `console.log('editor.mobile.spellcheckEnabled', Setting.value('editor.mobile.spellcheckEnabled'))`, we see that these are already `boolean`s:
```console
 LOG  editor.mobile.spellcheckEnabled true
```

Reply:
oh that's right, the Setting class makes the conversion. So it's fine then, thanks for checking